corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -1119,6 +1128,26 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
             self.isBlank = ko.computed(function () {
                 return !self.key() && !self.path();
             });
+            self.caseType = ko.computed(function () {
+                return self.action.case_type();
+            });
+            self.updatedDescription = ko.observable('');
```

Comment:
this is how the popover text gets updated after the inline edits (in the read function of `self.description`, and gets saved in the write line), otherwise it only updates on page refresh even after the description has been changed